spec/system/tag_synonyms_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+describe "Tag synonyms", type: :system, js: true do
+  let(:tags_page) { PageObjects::Pages::Tag.new }
+  fab!(:tag_1) { Fabricate(:tag, name: "design") }
+  fab!(:tag_2) { Fabricate(:tag, name: "art") }
+  fab!(:current_user) { Fabricate(:admin) }
+
+  before { sign_in(current_user) }
+
+  describe "when visiting edit tag page" do
+    it "should add existing tag as synonym" do
```

Comment:
```suggestion
    it "should allow admin to add existing tag as a synonym" do
```

Small suggestion to be clearer about the action we're testing for.